README.md

```diff
@@ -78,11 +78,11 @@ Packs could be either copied to a local directory, or downloaded from a remote l
 
 Copy the following files to `files/packs` directory.
 
-1. [WSO2 API Manager 4.0.0 package](https://wso2.com/api-management/install/)
+1. [WSO2 API Manager 4.1.0 package (.zipß)](https://github.com/wso2/product-apim/releases/tag/v4.1.0)
```

Comment:
Let's keep the original url here. Check other patterns as well.

Reply:
Updated